### DISCUSSION TITLE
[Feature Request] Make Blue-build's CLI more colorful.

### DISCUSSION THREAD

**Discussion by oomfiee** [blue-build/.github#34] [General]
i would like for Blue-build's CLI to be more colorful to make it easier to tell if your build failed or succeeded.

**Comment by xynydev** [comment]
@gmpinder transfer to CLI issues? I can't seem to be able to do that myself. 

Also need to make sure we support https://no-color.org/

**Reply by gmpinder** [reply]
The CLI already has color logs and will print red error on failure. So I don't see what else would need to be done. We can't exactly just go in and change the color of the Docker logs easily because we don't always know what the output is going to be.

Also, we already have mechanisms to obey the no color rules.

**Reply by oomfiee** [reply]
> The CLI already has color logs and will print red error on failure. So I don't see what else would need to be done. We can't exactly just go in and change the color of the Docker logs easily because we don't always know what the output is going to be.
> 
> Also, we already have mechanisms to obey the no color rules.

I see so CI logs are unsupported i guess

**Reply by xynydev** [reply]
I'll close this discussion then.

**Reply by gmpinder** [reply]
> I see so CI logs are unsupported i guess

What do you mean by this? The logs are available in your GitHub Actions and you can see what's happening in your build.

**Reply by oomfiee** [reply]
> > I see so CI logs are unsupported i guess
> 
> What do you mean by this? The logs are available in your GitHub Actions and you can see what's happening in your build.

I meant the option to make the CI logs to be more colorful

**Reply by gmpinder** [reply]
I'm still unclear as to what exactly you want colored. We make calls to Docker/Podman to build the image. There's no simple way to easily pick out the part of the Docker logs that contributed to the error or the failure of the build. We simply throw an error log saying that the build failed and the build log is there above the error.
